### ISSUE TITLE
TensorBoard not showing Complete data

### ISSUE THREAD

**Issue by nikhilmaram** [tensorflow/tensorboard#1154]
TensorBoard version : 1.6.0
OS version : Mac OS High Sierra
Python version : 3.6

Issue:
1. I am running my code through 1000 epochs and I am writing to TensorBoard each epoch,But in TensorFlow I am not able to see values corresponding to all 1000 epochs. I only see for 300 epochs(x-axis is only 300).
2. If I run my code for less than 200 epochs. I don't see any scalar values in my TensorBoard. I can only see the graph.

Attached are the code snippets : part1.png, part2.png. 

After running the code. I save summaries in 3 different directories (train, test,val). All these directories are saves in same directory.

Used this command to view this TensorBoard : tensorboatd --logdir=./ --port=6009.

TensorBoard-1000.png : TensorBoard when code is run for 1000 epochs. You can see x-axis till 300.
TensorBoard-200.png: TensorBoard when code is run for 200 epochs. There are no scalars.

<img width="1013" alt="part1" src="https://user-images.githubusercontent.com/23695133/39290527-be65e46c-48e4-11e8-9213-68836c75d298.png">
<img width="1011" alt="part2" src="https://user-images.githubusercontent.com/23695133/39290528-be7ce428-48e4-11e8-9492-267a239f92fe.png">
<img width="1438" alt="tensorboard-200" src="https://user-images.githubusercontent.com/23695133/39290533-c358357e-48e4-11e8-9f02-6c96726618f5.png">
<img width="979" alt="tensorboard-1000" src="https://user-images.githubusercontent.com/23695133/39290535-c38207b4-48e4-11e8-8ab5-2d4715f2b2a2.png">

**Comment by nfelt** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it isn't clearly a TensorBoard bug.  There is a larger community available there.

If you're able to reduce this problem to a minimal reproduction and have confirmed that it isn't an issue with only your local system or with the data in the event file itself, feel free to reopen the issue.

**Comment by zacharyyamaoka** [comment]
I had the same issue: Try closing reloading the tensorboard plugin. Seems that it needs to be refreshed to read the latest data.

In Jupyter Notebook:
`!kill 12486 #Change this value to the process id for tensorboard on your computer
%tensorboard --logdir logs/fit`
